collect_diagnosis_data_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2025 ScyllaDB
+
+from sdcm.tester import ClusterTester
+from sdcm import nemesis
+
+
+class ScyllaDiagnosisReport(ClusterTester):
```

Comment:
usually we don't add new tests - only in specific cases. 
For this should utilize 'individual nemesis' case